### ISSUE TITLE
What URLs to use for help pages?

### ISSUE THREAD

**Issue by gvwilson** [gvwilson/tidyblocks#370]
Should all help pages have uniform URLs or language-specific URLs? I.e., should the Spanish help page for "data" be `/es/datos/` or `/es/data/`?

-   Pro: URLs will be easier to remember if they are translated.
-   Con: We will then have to do translation for all blocks' `helpUrl` fields when constructing them.

The same question comes up with (for example) the glossary: should the Spanish version be `/es/glosario/` or `/es/glossary/`? The more differences there are, the more maintenance we're going to have to do, but using English terms for the links seems inappropriate.

**Comment by elssam** [comment]
Hi Greg, I think it's more helpful to have the translated version of the glossary. Yes, including translated words to the URLs that lead to it.

**Comment by demar01** [comment]
I agree. Similar issue is on the name of the columns ( e.g red, blue, green on colores), which is then it is another layer to translate...
